air/src/execution_step/air/call/resolved_call.rs

```diff
@@ -129,7 +130,11 @@ impl<'i> ResolvedCall<'i> {
             tetraplets,
         } = self.resolve_args(exec_ctx)?;
 
-        let serialized_tetraplets = serde_json::to_string(&tetraplets).expect("default serializer shouldn't fail");
+        let serialized_tetraplets = {
+            let span = tracing::span!(tracing::Level::INFO, "serde_json::to_string(tetraplets)");
```

Comment:
Such construction now exist in few places, and it looks like a great application for a macros.  I haven't found proper place for it.

Reply:
Yep, I'd agree that it's a good application for macros.